data/ui/ButtonRow.ui

```diff
@@ -9,10 +9,10 @@
       <object class="GtkBox">
         <property name="visible">True</property>
         <property name="can_focus">False</property>
-        <property name="margin_left">6</property>
-        <property name="margin_right">6</property>
-        <property name="margin_top">4</property>
-        <property name="margin_bottom">4</property>
```

Comment:
The thing is that they don't look the same with these values, which is why I tried to increase them. If you compare the original to the control-center's keyboard panel, this is what you see:

![screenshot from 2017-08-21 11-18-05](https://user-images.githubusercontent.com/1260727/29512438-7c002f1e-8662-11e7-87bb-c6446af3dff2.png)

versus

![screenshot from 2017-08-21 11-18-19](https://user-images.githubusercontent.com/1260727/29512444-83c8ba68-8662-11e7-8f92-0bf01bec8a62.png)

Compare that to the new values:

![screenshot from 2017-08-21 11-20-15](https://user-images.githubusercontent.com/1260727/29512531-ba8373fe-8662-11e7-9b99-7b964ed9a7ad.png)

According to GtkInspector, the new values give an allocation of 35 pixels in height versus 38 in the keyboard panel so it's still not exactly the same. I suppose their height is larger due to the image.